cmd/vulcan-spf/main.go

```diff
@@ -15,7 +15,7 @@ var (
 )
 
 func main() {
-	run := func(ctx context.Context, target string, optJSON string, state state.State) (err error) {
+	run := func(ctx context.Context, target, assetType, optJSON string, state state.State) (err error) {
```

Comment:
Shouldn't we check if the asset is reachable?

Reply:
Done, same as DKIM/DMARC.